transaction/merkletreeparent.go

```diff
@@ -0,0 +1,54 @@
+package transaction
+
+import (
+	"encoding/hex"
+
+	"github.com/bitcoin-sv/go-sdk/chainhash"
+	crypto "github.com/bitcoin-sv/go-sdk/primitives/hash"
+	"github.com/bitcoin-sv/go-sdk/util"
+)
+
+// MerkleTreeParentStr returns the Merkle Tree parent of two Merkle
+// Tree children using hex strings instead of just bytes.
+func MerkleTreeParentStr(leftNode, rightNode string) (string, error) {
+	l, err := hex.DecodeString(leftNode)
+	if err != nil {
+		return "", err
+	}
+	r, err := hex.DecodeString(rightNode)
+	if err != nil {
+		return "", err
+	}
+
+	return hex.EncodeToString(MerkleTreeParent(l, r)), nil
+}
+
+// MerkleTreeParent returns the Merkle Tree parent of two Merkle
+// Tree children.
```

Comment:
Consider putting the comment in one line because `Merkle` .... `Tree children` looks weird.

Reply:
Done